crates/ruma-client-api/src/user_directory/search_users.rs

```diff
@@ -0,0 +1,109 @@
+//! `POST /_matrix/client/*/user_directory/search`
+
+pub mod v3 {
+    //! `/v3/` ([spec])
+    //!
+    //! [spec]: https://spec.matrix.org/v1.2/client-server-api/#post_matrixclientv3user_directorysearch
+
+    use js_int::{uint, UInt};
+
+    use ruma_api::ruma_api;
+
+    use ruma_identifiers::{MxcUri, UserId};
+
+    use serde::{Deserialize, Serialize};
```

Comment:
There weren't any empty lines between these imports before, what happened?

Reply:
Yes. You can set [group_imports](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#group_imports) to `StdExternalCrate` in `.rustfmt.toml` and then undo any invalid changes (that setting is buggy in that it will move imports from modules together with ones from external crates).